.github/workflows/standards-and-tests.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       run: bash bin/install-wp-tests.sh wordpress_test root root localhost ${{ matrix.wordpress }}
 
     - name: Run PHP Tests
-      run: composer test
+      run: composer test -- --exclude=integrations
```

Comment:
not happy about this, but as we discussed it is a temporary fix.

Reply:
Are we going to ignore integrations only for `experimental` pipelines?